packages/ui-select/src/Select/index.tsx

```diff
@@ -520,7 +522,7 @@ class Select extends Component<Props> {
                 // @ts-expect-error ts-migrate(2345) FIXME: Argument of type 'null' is not assignable to param... Remove this comment to see the full error message
                 const index = this._optionIds.indexOf(highlightedOptionId)
                 highlightId =
-                  index > -1 ? this._optionIds[index + direction] : null
+                  index > -1 ? this._optionIds[index + direction!] : null
```

Comment:
Hm, as far as I can tell, direction is not always provided, more like... it needs EITHER and id OR a direction. Probably not worth typing it correctly now, but could you please leave a comment for later?

Reply:
done